README.md

```diff
@@ -15,11 +12,11 @@ We encourage installing Qiskit Machine Learning via the pip tool (a python packa
 pip install qiskit-machine-learning
 ```
 
-**pip** will handle all dependencies automatically and you will always install the latest
+**pip** will handle all dependencies automatically and will always install the latest
```

Comment:
Lets stay with the old text - this is how the other repos, eg Terra talk about this